File: client/src/models/FileUploadResponse.ts
import { MailAddressParams } from './MailAddress'

class FileUploadResponse {
  total: number
  channel: string
  mailAddressList: MailAddressParams[]

  constructor(
    total: number,
    channel: string,
    mailAddressList: MailAddressParams[]
  ) {
    this.total = total
    this.channel = channel
    this.mailAddressList = mailAddressList
  }
}

export default FileUploadResponse

File: Spike.AWS.FileUpload.API/Models/FileUploadResponse.cs
using System.Collections.Generic;

namespace Spike.AWS.FileUpload.API.Models
{
    public class FileUploadResponse
    {
        public int Total => MailAddressList.Count;

        public string Channel { get; set; }

        public string DncLists { get; set; }

        public string Notes { get; set; }

        public List<MailAddress> MailAddressList { get; set; } =  [];
    }
}